helm/charts/kratos/templates/deployment.yaml

```diff
@@ -187,7 +187,7 @@ spec:
           readinessProbe:
             {{- toYaml .Values.deployment.readinessProbe | nindent 12 }}
           resources:
-            {{- toYaml .Values.resources | nindent 12 }}
+            {{- toYaml .Values.deployment.resources | nindent 12 }}
```

Comment:
Hi again, sorry for the delay but I needed a moment to think about this. 
You are right, right now this is a change, I would not call it breaking, but it will require user input to change the chart values. 
However, we can fix it in a more smooth way:

```
if .Values.resources
  use them
else if .Values.deployment.resources
  use them
else {}
```

Could you update your PR with such a model?

Reply:
Hi, ok, I see. The idea is to support `resources` both at `root` level or at `deployment` level. 

Is it something that you want to support in long term or you plan to deprecate the specification at `root` level and remove it someday ?

Reply:
In my point of view, if we have something in `.Values.deployment.resources` it must have precedence on `.Values.resources`. What do you think ?

Reply:
I think it would make more sense to deprecate the root level further on. The idea of root level resources was ok when we had only 1 application/deployment in the charts. Since now we have included jobs and others, i think it may be confusing as to what those options relate to

Reply:
Ok, that what I understood 👍‍